pandas/core/generic.py

```diff
@@ -2776,6 +2776,7 @@ def to_latex(
         multirow=None,
         caption=None,
         label=None,
+        formatters_col=None,
```

Comment:
In line what @jreback was referring to, this should probably be `float_format` that mirrors the kwarg in `to_csv`

Reply:
I think I've addressed with the name change of the variable? If not let me know.